Dockerfile

```diff
@@ -13,6 +13,7 @@ ENV RAYGUN_REACT_CLIENT_API_KEY=${RAYGUN_REACT_CLIENT_API_KEY}
 FROM base as production_build
 ENV NODE_ENV=production
 COPY . .
+COPY robots.txt.prod public/robots.txt
```

Comment:
This will happen on all deployments - alpha, beta, live etc.  as they are all production environments. (as opposed to dev or test).  Dev is only localhost and test is only when running npm test.   So this option won't work